src/main/webapp/i18n/de/quizExercise.json

```
@@ -61,7 +61,8 @@
             "isOpenForPractice": "Zum Üben freigegeben",
             "isPlannedToStart": "Geplante Startzeit",
             "openForPractice": "Zum Üben Freigeben",
-            "startNow": "Jetzt Starten",
+            "startQuiz": "Quiz Starten",
+            "startBatch": "Ladung Starten",
```

To match the existing translations
```suggestion
            "startBatch": "Gruppe Starten",
```